src/test/java/com/metamx/emitter/core/HttpEmitterConfigTest.java

```diff
@@ -44,20 +39,28 @@ public void testDefaults()
   }
 
   @Test
-  public void testSettingEverything()
+  public void testSettingEverything() throws Exception
   {
-    final Properties props = new Properties();
-    props.setProperty("com.metamx.emitter.flushMillis", "1");
-    props.setProperty("com.metamx.emitter.flushCount", "2");
-    props.setProperty("com.metamx.emitter.http.url", "http://example.com/");
-    props.setProperty("com.metamx.emitter.http.basicAuthentication", "a:b");
-    props.setProperty("com.metamx.emitter.http.batchingStrategy", "newlines");
-    props.setProperty("com.metamx.emitter.http.maxBatchSize", "4");
-    props.setProperty("com.metamx.emitter.http.maxBufferSize", "8");
-    props.setProperty("com.metamx.emitter.http.flushTimeOut", "1000");
-
     final ObjectMapper objectMapper = new ObjectMapper();
-    final HttpEmitterConfig config = objectMapper.convertValue(Emitters.makeHttpMap(props), HttpEmitterConfig.class);
```

Comment:
that is true.
does the other code you have that creates object doesn't do it the way druid does and instead relies on the property transformations in Emitters.makeHttpMap?